discord/state.py

```diff
@@ -1320,6 +1321,15 @@ def parse_guild_delete(self, data) -> None:
         self._remove_guild(guild)
         self.dispatch("guild_remove", guild)
 
+    def parse_guild_audit_log_entry_create(self, data) -> None:
+        guild = self._get_guild(int(data["guild_id"]))
+        user = self.get_user(data["user_id"])
```

Comment:
Potentially make a raw event as it depends on user cache state, or resolve user to something like `Object`

Reply:
As in make a raw event and a normal event?

Reply:
Nevermind, I get it now.